ros_ws/src/arudino/motor/motor.ino

```diff
@@ -108,6 +108,10 @@ void velocityCallback(const std_msgs::Float64& msg)
 
 void setup() 
 {
+    //Fix the Motor Whine
+    //After testing on IGVC 8 gave the best results
+    set_pwm_frequency(8);
```

Comment:
Maybe have this as a const or define so we can just change it at the top of the file and that way it looks less like a magic number.